docs/working-with-projects/working-with-images/preheat-images.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Preheat Images
+weight: 30
+---
+Preheat action is policy-based driven way, and the scope of preheat policy is limited in the project level. Preheat 
+policy created under the specified project can only preheat images manged under that project.
+
+## Create Preheat Policy
+
+To preheat images, you need to create preheat policy first.
+
+1. Go to **Projects** and open your project from the project list.
+1. Open the **P2P Preheat** tab and then click **+ NEW POLICY** button to open the policy creation dialog.
+  ![policy creation dialog](../../../img/p2p-preheat/policy-creation-dialog.png)
+1. For the **Provider**, select one pre-configured preheat provider instance as target from the dropdown list.
+1. Input a suitable name and description(optional) to identify and describe your creating policy.
+1. Set the repository filter(required) by following the [doublestar pattern](https://github.com/bmatcuk/doublestar#patterns) to 
+narrow down the scope of preheating images.
+1. Set the tags filter(required) by following the same [doublestar pattern](https://github.com/bmatcuk/doublestar#patterns) to 
+determine images with what tags can be considered to be preheated.
```

Comment:
i would remove 'to determine images with what tags can be considered to be preheated.'  It's redundant